kube/manifest/service-proxima-do.yaml

```
@@ -0,0 +1,82 @@
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb-ip
+    service.beta.kubernetes.io/aws-load-balancer-internal: "false"
+  labels:
+    app: starcoin
+    network: proxima
+    statefulset.kubernetes.io/pod-name: starcoin-0
+  name: starcoin-0
+  namespace: starcoin-proxima
+spec:
+  ports:
+  - port: 9840
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)